cli/cluster.go

```diff
@@ -85,3 +93,17 @@ func runClusterRemove(args *docopt.Args) error {
 
 	return nil
 }
+
+func runClusterDefault(args *docopt.Args) error {
+	name := args.String["<cluster-name>"]
+
+	if config.SetDefault(name) {
```

Comment:
Flip the conditional, so that we avoid indenting code:

```go
if !config.SetDefault(name) {
   return nil
}

if err := config.SaveTo(configPath()); err != nil {
   return err
}
...
```